include/openssl/core_names.h

```diff
@@ -154,6 +154,11 @@ extern "C" {
 #define OSSL_KDF_NAME_KRB5KDF       "KRB5KDF"
 
 /* PKEY parameters */
+/* Common PKEY parameters */
+#define OSSL_PKEY_PARAM_BITS                "bits" /* integer */
+#define OSSL_PKEY_PARAM_SIZE                "size" /* integer */
```

Comment:
I would leave this as a thought, that could be discussed in a separate PR.

In the interface with the provider we are not forced to use the same name of the EVP layer interface, especially in cases where we agree the original naming is quite unfortunate and misleading. 

What about renaming this as OSSL_PKEY_PARAM_OUT_MAX_SIZE or something similar to overcome the poor naming of the EVP API?

Reply:
I would say that `OSSL_PKEY_PARAM_MAX_SIZE` is good enough, with "max-size" as the string